website/docs/cloud-docs/agents/index.mdx

```diff
@@ -127,6 +127,23 @@ Agents are organized into pools, which can be managed in Terraform Cloud's UI. E
 
    ![Screenshot: Unable to delete an agent pool with attached workspace](/img/docs/agent-delete-invalid.png)
 
+### Scope an Agent Pool to Specific Workspaces
+
+Scoping an agent pool lets you specify which workspaces are allowed to use it. The agent pool will then only be available from within the chosen workspaces; it will not appear other workspace's lists of available agent pools. Scoping an agent pool does not remove it from workspaces that are actively using it. To remove access, you must first remove the agent pool from within the workspace's settings.
+To scope an agent pool to specific workspaces within the organization:
```

Comment:
```suggestion
Scoping an agent pool lets you specify which workspaces are allowed to use it. The agent pool will then only be available from within the chosen workspaces; it will not appear other workspace's lists of available agent pools. Scoping an agent pool does not remove it from workspaces that are actively using it. To remove access, you must first remove the agent pool from within the workspace's settings.

To scope an agent pool to specific workspaces within the organization:
```